docs/Core.rst

```diff
@@ -1,152 +1,197 @@
+.. _dfhack-core:
+
 ###########
 DFHack Core
 ###########
 
+DFHack commands can be implemented in three ways, all of which
+are used in the same way:
+
+:builtin:   commands are implemented by the core of DFHack, and quite rare.
```

Comment:
This was more a comment about the wording - "less common", maybe, but not "quite rare"